controllers/workflowdef/image.go

```diff
@@ -23,8 +23,8 @@ import (
 const (
 	latestImageTag              = "latest"
 	nightlySuffix               = "nightly"
-	defaultWorkflowDevModeImage = "quay.io/kiegroup/kogito-swf-devmode"
-	defaultWorkflowBuilderImage = "quay.io/kiegroup/kogito-swf-builder"
+	DefaultWorkflowDevModeImage = "quay.io/kiegroup/kogito-swf-devmode"
+	DefaultWorkflowBuilderImage = "quay.io/kiegroup/kogito-swf-builder"
 )
```

Comment:
I'd rather not make these constants public

Reply:
Fix: https://github.com/kiegroup/kogito-serverless-operator/pull/172